src/domain/forms/components/Contact.tsx

```diff
@@ -17,7 +16,7 @@ interface Props<T> {
   children: React.ReactNode;
 }
 
-const Contact = <T extends unknown>({
+const Contact = <T,>({
```

Comment:
Is the comma a typo or is it really needed?

Reply:
Yes, it's needed.